sonaranalyzer-dotnet/rspec/vbnet/Sonar_way_recommended_profile.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What's that?

Reply:
I have no idea. I just ran `.\scripts\rspec\rspec.ps1 vbnet`. But I removed it as we don't know exactly what it is